src/App.js

```diff
@@ -1,13 +1,18 @@
-import "./App.css";
+import "./Styles/App.css";
+import "./Styles/Feedpost.css";
 import { Routes, Route } from "react-router-dom";
 
 import HomePage from "./Pages/Homepage";
+import FeedPage from "./Pages/FeedPage";
+import SignUp from "./Pages/SignUp";
 
 function App() {
   return (
     <div className="App">
       <Routes>
-        <Route path="/" element={HomePage} />
+        <Route path="/" element={<HomePage />} />
+        <Route path="/Feed" element={<FeedPage />} />
```

Comment:
Not sure we ever mentioned this, but URLs should be lowercase, actually, so `/feed` not `/Feed`.